pyproject.toml

```diff
@@ -19,7 +19,7 @@ dependencies = [
     'gwcs >=0.18.1',
     'jsonschema >=4.8',
     'numpy >=1.22',
-    'photutils >=1.6.0',
+    'photutils @ git+https://github.com/astropy/photutils.git',
```

Comment:
It looks to me like this is likely out now?

Reply:
The last remaining and necessary addition to photutils is from https://github.com/astropy/photutils/pull/1617. I'll ask @larrybradley if the next release is likely to be soon.